.clabot

```diff
@@ -63,6 +63,8 @@
     "FedericoCaruso",
     "njokuScript",
     "Calla-Ji",
+    "TylerSehr",
```

Comment:
Please do these changes in *another* PR.

Reply:
Opened another PR with just these changes:
https://github.com/unlock-protocol/unlock/pull/13537